pkg/apis/ext.cattle.io/v1/types.go

```diff
@@ -1,24 +1,29 @@
 // +kubebuilder:skip
 package v1
 
-// This file should contain type definitions for the ext.cattle.io/v1 API group.
-// Below is an example of what it could look like. This comment can be removed once
-// types have been added to this package.
-//
-//	import (
-// 		metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-// 	)
-//
-// 	// +genclient
-// 	// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-//
-// 	type TestType struct {
-// 		metav1.TypeMeta   `json:",inline"`
-// 		metav1.ObjectMeta `json:"metadata,omitempty"`
-//
-// 		Spec TestTypeSpec `json:"ok"`
-// 	}
-//
-// 	type TestTypeSpec struct {
-// 		Enabled bool `json:"enabled"`
-// 	}
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// User Activity
+type UserActivity struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec is the spec of UserActivity
+	Spec UserActivitySpec `json:"spec"`
+	// +optional
+	Status UserActivityStatus `json:"status"`
+}
+
+type UserActivitySpec struct {
+	TokenId string `json:"tokenId"`
+}
+
+type UserActivityStatus struct {
+	CurrentTimeout string `json:"currentTimeout"`
```

Comment:
The fields in spec and status should have comments explaining them, i.e. their purpose and usage.
See the token PR for examples. The comments become the field descriptions in the generated file
`./pkg/generated/openapi/zz_generated_openapi.go`, and thus available to users via `kubectl explain`.
IOW these comments are not just internal developer documentation, but also public user documentation.